### ISSUE TITLE
PM25 measurement

### ISSUE THREAD

**Issue by gaguevaras** [Kels316/DIY-Air-Purifier#3]
If you're interested in getting another KPI to track how effective the air purifier is, you could easily build this PM25 sensor:

https://www.hackster.io/canairio-guide-team/canairio-citizen-network-for-air-quality-monitoring-bbf647

It would allow you to measure the effect your purifier has on indoor air quality by measuring how many particles under 2.5 microns per cubic meter there are before and after operating it. Good luck!

**Comment by Mihaylov93** [comment]
Also there is the MQ sensor series like https://www.olimex.com/Products/Components/Sensors/Gas/SNS-MQ135/resources/SNS-MQ135.pdf

**Comment by Kels316** [comment]
yes I have considered building a test bench for this, but time has been short recently. Definitely on the to-do list for the coming months.
